quickwit/quickwit-proto/protos/quickwit/search_api.proto

```diff
@@ -226,16 +230,13 @@ message PartialHit {
   // - the split_id,
   // - the segment_ord,
   // - the doc id.
-  oneof sort_value {
-    uint64 u64 = 5;
-    int64 i64 = 6;
-    double f64 = 7;
-    bool boolean = 8;
-  }
+
   // Deprecated
   reserved 1;
   // Room for eventual future sorted key types.
-  reserved 9 to 20;
+  reserved 12 to 20;
+  SortBy sort_value = 10;
```

Comment:
can we call the property sort_by too?

Reply:
(There is also a SortValue thing that makes everything confusing)

Reply:
I renamed it to `SortByValue` as SortBy is not really accurate. It's unnecessary, but I don't know how to get rid of it with prost